docs/website/docs/walkthroughs/create-a-pipeline.md

```diff
@@ -2,4 +2,142 @@
 sidebar_position: 1
 ---
 
-# Create a pipeline
\ No newline at end of file
+# Create a pipeline
+
+Follow the steps below to create a [pipeline](./glossary.md/#pipeline) from the Twitter API to 
+Google BigQuery from scratch. The same steps can be repeated for any source and destination of your 
+choice--use `dlt init <source> <destination>` and then build the pipeline for that API instead.
+
+Please make sure you have [installed `dlt`](./installation.mdx) before getting started here.
+
+## 1. Initialize project
+
+Create a new empty directory for your `dlt` project by running
+```
+mkdir twitter-bigquery
+```
+
+Start a `dlt` project with a pipeline template that loads data to Google BigQuery by running
+```
+dlt init twitter bigquery
+```
+
+Install the dependencies necessary for Google BigQuery:
+```
+pip install -r requirements.txt
+```
+
+## 2. Add Google BigQuery credentials
+
+You can follow the steps in [Set up Google BigQuery](../getting-started.md#2-set-up-google-bigquery)
+to create the service account credentials you'll need for BigQuery and add them to `.dlt/secrets.toml`.
+
+## 3. Add Twitter API credentials
+
+You will need to [sign up for the Twitter API](https://developer.twitter.com/en/docs/platform-overview)
+and create a project to grab your Bearer Token for the Twitter API.
+
+Copy the value of the bearer token into `.dlt/secrets.toml`
+```
+[sources]
+
+twitter_bearer_token = '<bearer token value>'
```

Comment:
Here I'd mention that this value must correspond to the name of the argument here:
`twitter_bearer_token=dlt.secrets.value`
in `def twitter_resource(api_url=dlt.config.value, twitter_bearer_token=dlt.secrets.value):`

and if they want to change the name of the secret they also must change the name of the argument (and vice versa)